lib/index.js

```diff
@@ -3,5 +3,14 @@ const { basename, dirname } = require('path')
 const getName = (parent, base) =>
   parent.charAt(0) === '@' ? `${parent}/${base}` : base
 
-module.exports = dir => dir ? getName(basename(dirname(dir)), basename(dir))
-  : false
+module.exports = (dir) => {
+  if (!dir) {
```

Comment:
Fun fact: removing ternary operators is a good way to get on my good side ;)

Reply:
Good to know ;)